scijava/scijava-javadoc-parser/src/main/java/org/scijava/javadoc/parser/RuntimeJavadocHelper.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2015 David Nault and contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
We should rewrite the LICENSE here - @ctrueden is there a protocol for doing so?

Reply:
You can model it after `Types.java`:

1. Put a commented notice about the original license after our initial license block; and
2. Make sure that commented notice tells people to check the `NOTICE`/`NOTICE.txt` for additional details.

As an aside, that `Types.java` needs to be adjusted to point to an actually existing file called either `NOTICE` or `NOTICE.txt` or `THIRDPARTY.txt`. See [scijava-common's `NOTICE.txt`](https://github.com/scijava/scijava-common/blob/scijava-common-2.96.0/NOTICE.txt).

Reply:
Added with 0fa1d47d